src/plugins/ncr/sagas/forceSaveAfterSuccessFlow.js

```diff
@@ -0,0 +1,57 @@
+import { put, select } from 'redux-saga/effects';
+import { blur, getFormValues } from 'redux-form';
+import isEqual from 'lodash/isEqual';
+import swal from 'sweetalert2';
+import formData from '@triniti/cms/plugins/news/utils/formData';
```

Comment:
@hunyoboy wouldnt want to have an ncr saga import a news util. i would probably make it a standard util in `src/utils/` and then pass it in through config as `getFormData: (key) => formData.get(key)`